Library/Homebrew/cmd/untap.rb

```diff
@@ -1,12 +1,11 @@
-# typed: strict
+# typed: true
 # frozen_string_literal: true
 
 require "abstract_command"
-require "untap"
 
 module Homebrew
   module Cmd
-    class UntapCmd < AbstractCommand
+    class Untap < AbstractCommand
```

Comment:
Let's keep the name the same as it was before unless we're changing all the other commands to no longer have the `Cmd` suffix.

Reply:
Only a select few commands use the magic suffix, to avoid name resolution errors. It's appropriate to remove it when the name is no longer ambiguous.

Reply:
Gotcha, ignore me then!